test/framework/toolchain.py

```diff
@@ -870,9 +870,16 @@ def test_old_new_iccifort(self):
         libblas_mt_intel3 += " -Wl,--end-group -Wl,-Bdynamic -liomp5 -lguide -lpthread"
 
         # no -lguide
+        libblas_intel4 = "-Wl,-Bstatic -Wl,--start-group -lmkl_intel_lp64 -lmkl_sequential -lmkl_core"
+        libblas_intel4 += " -Wl,--end-group -Wl,-Bdynamic"
         libblas_mt_intel4 = "-Wl,-Bstatic -Wl,--start-group -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core"
         libblas_mt_intel4 += " -Wl,--end-group -Wl,-Bdynamic -liomp5 -lpthread"
 
+        libfft_intel4 = libblas_intel4.replace('-lmkl_intel_lp64', '-lfftw3xc_intel -lmkl_intel_lp64')
+        libfft_mt_intel4 = libblas_mt_intel4.replace('-lmkl_intel_lp64', '-lfftw3xc_intel -lmkl_intel_lp64')
+        libfft_mt_intel4 = "-Wl,-Bstatic -Wl,--start-group -lfftw3xc_intel -lmkl_intel_lp64 -lmkl_intel_thread"
+        libfft_mt_intel4 += " -lmkl_core -liomp5 -lpthread -Wl,--end-group -Wl,-Bdynamic"
```

Comment:
These two lines doesn't make sense, you're overwriting the value you just set libfft_mt_intel4 to.

Reply:
Hmmm, I think the replace would be more correct? I.e. just as libfft_intel4 uses replace on libblas, libfft_mt_intel4 should use replace on libblas_mt ?? Shorter and the -liomp5 -lpthread should be outside the grouping as far as I can see from the code in toolchain/fft.py above

Reply:
Agreed, and fixed in 269d6ef